_data/changelogs/component-file-input.yml

```diff
@@ -2,6 +2,12 @@ title: File input
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Removed disabled file input variant preview.
+    summaryAdditional: 
+    githubPr: 2114
+    githubRepo: uswds-site
+    versionUswds: 3.5.0
```

Comment:
@amyleadem Added changelog item here! I was on the fence about adding `summaryAdditional` clarifying something like:

> Our work to normalize disabled form elements caused us to remove the `.json` file required for component preview.

But was worried it was unnecessary.

We can also consider adding these previews back once we get finish our disabled form guidance.

Reply:
I think you made the right call. I don't see a need to add more details.